go.mod

```diff
@@ -154,6 +154,12 @@ require (
 	nhooyr.io/websocket v1.8.6 // indirect
 )
 
+// Below are the short-lived replace of the Cosmos SDK
+replace (
+	cosmossdk.io/core => ./core
+	cosmossdk.io/x/tx => ./x/tx
```

Comment:
What's the process here to remove short-lived replaces? Should I tag a version right after, or wait a bit (given e.g. there are a lot of other changes in x/tx these days).

Reply:
yes, we remove these replaces by tagging a version from what I know. But IDK when we do that

Reply:
just curious to know why are we using these replace directives here and other go mods ?

Reply:
> just curious to know why are we using these replace directives here and other go mods ?

Here are my steps to why I added them:
- I replaced all `any` with `anyutil` in go code
- it happened in sdk, core and x/tx
- I bumped to beta.3 in sdk, core, and x/tx
- I got an error saying that sdk imports core, which imports cosmos-proto (beta.2), which doesn't have anyutil
- I added this replace and it worked.

Reply:
> Actually lately (for log, collections and math) we've been using pseudo version of the latest commit from main / the branch.

Should I switch to a pseudo version of the latest commit of this branch? Not sure which is best compared to replace (I prefer the latter)

Reply:
I believe the reason with the first, and that it's one less PR (when we need not to miss a replace), and dependabot takes care of the rest if it's a pseudo version when we tag something.